Resources/Prototypes/Language/genericlanguages.yml

```diff
@@ -3,6 +3,7 @@
 # with the Xenoglossy psionic power, as well as Ghosts, and AGhosts.
 - type: language
   id: Universal
+  isVisibleLanguage: true
```

Comment:
Should Universal be visible? Is it really a language, or just a kinda magical thing that's understandable by everyone?